docs/api/orders.rst

```diff
@@ -3,165 +3,89 @@ Orders API
+| adjustments           | List of adjustments related to cart         |
++-----------------------+---------------------------------------------+
+| adjustments_total     | Sum of all order adjustments                |
++-----------------------+---------------------------------------------+
+| total                 | Sum of items total and adjustments total    |
++-----------------------+---------------------------------------------+
+| customer              | `Customer detailed serialization`_ for cart |
++-----------------------+---------------------------------------------+
+| channel               | `Default channel serialization`_            |
++-----------------------+---------------------------------------------+
+| currency_code         | Currency of a cart                          |
++-----------------------+---------------------------------------------+
+| checkout_state        | `State of checkout process`_                |
++-----------------------+---------------------------------------------+
+| state                 | `One of available order states`_            |
++-----------------------+---------------------------------------------+
+| checkout_completed_at | Date when the checkout has been completed   |
++-----------------------+---------------------------------------------+
+| number                | Number of an order                          |
++-----------------------+---------------------------------------------+
+| shipping_address      | Detailed address serialization              |
++-----------------------+---------------------------------------------+
+| billing_address       | Detailed address serialization              |
++-----------------------+---------------------------------------------+
+| shipments             | Detailed shipment serialization             |
++-----------------------+---------------------------------------------+
+| payments              | Detailed payment serialization              |
++-----------------------+---------------------------------------------+
+
+.. _Customer detailed serialization: http://docs.sylius.org/en/latest/api/customers.html#customers-api
+.. _Default channel serialization: http://docs.sylius.org/en/latest/api/channels.html#channels-api
+.. _One of available order states: http://docs.sylius.org/en/latest/components/Order/state_machine.html#order-states
+.. _State of checkout process: http://docs.sylius.org/en/latest/book/orders/checkout.html?highlight=checkout%20states#checkout-state-machine
 
-You can retrieve the full list order by making the following request:
-
-.. code-block:: text
-
-    GET /api/v1/orders/
-
-Parameters
-~~~~~~~~~~
-
-page
-    Number of the page, by default = 1
-limit
-    Number of items to display per page
+Getting a single order
+----------------------
 
-Response
-~~~~~~~~
+You can request detailed order information by executing the following request:
 
-The response will contain the newly created order information.
+Definition
+..........
```

Comment:
I just use the same formatting as in [shipping categories](http://docs.sylius.org/en/latest/api/shipping_categories.html). We can change it, but in separate PR and in all API docs.